noxfile.py

```diff
@@ -60,6 +60,12 @@ class = StreamHandler
         sa.Column('id', sa.Integer, primary_key=True),
         sa.Column('name', sa.String(50), nullable=False),
         sa.Column('description', sa.Unicode(200)),
+    )
+    op.alter_column(
+        'account',
+        'name',
+        existing_type=sa.String(50),
+        nullable=True,
```

Comment:
Adding the issue case into the migration test (the original error could be repeated this way before these changes)